### PR TITLE
Fix: Nested tabs - Typography hover color is not working [ED-8842]

### PR DESCRIPTION
**Pull request by davseve**

### CODE REVIEW

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js [hunk 1]
```diff
@@ -207,6 +207,39 @@ test.describe( 'Nested Tabs tests', () => {
...
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Add widgets.
+		await editor.addWidget( 'nested-tabs', container );
+		await editor.getPreviewFrame().waitForSelector( '.e-n-tab-title.e-normal.e-active' );
+
+		// Act.
+		await editor.activatePanelTab( 'style' );
+		await page.locator( '.elementor-control-section_title_style' ).click();
```
Comment (hein-obox) [inline tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:229]: @davseve There is a background color selector inside the `editor-page.js` file.

Comment (davseve) [reply tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:229]: Hi, `setBackgroundColor()` sets the background color.
Here I choose a color and not the background color.
But great function :) .
If I didn't understand you well, let's talk about that.

**Review by hein-obox — APPROVED** [review]

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js (hunk 1 above)
Comment (hein-obox) [reply tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:229]: @davseve My mistake. You are correct.

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js [hunk 2]
```diff
@@ -207,6 +207,39 @@ test.describe( 'Nested Tabs tests', () => {
 		await expect( activeTab ).toHaveCSS( 'margin-bottom', '50px' );
 		await expect( lastTab ).toHaveCSS( 'margin-top', '25px' );
 	} );
+
+	test( 'Check that active and non-active tabs color changes on hover', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+
+		await wpAdmin.setExperiments( {
```
Comment (hein-obox) [inline tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:215]: @davseve Comment from Tzvi: can we set the experiments to inactive in the end. 
Addition from me: you can also create `afterAll()` and `beforeAll()` functions.

Comment (davseve) [reply tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:215]: Done, 
Regarding the `afterAll()` and `beforeAll()`, we can add it in the next task.

**Review by TzviRabinovitch — APPROVED** [review]

**Pull request merged**